AAEmu.Game/Models/Game/Char/Character.cs

```diff
@@ -1484,6 +1504,39 @@ public static Character Load(uint characterId)
                 return Load(connection, characterId);
         }
 
+        public uint MaxBreath
+        {
+            get
+            {
+                if (Race == Race.Elf)
+                    return 80000;
+                else
+                    return 60000;
+            }
+        }
```

Comment:
This should be UnitAttribute "LungCapacity" (91) 

Once we add passives it'll even compute it by default, aswell as fix any buff adding to it (there are a few items & buffs adding breathing capacity)

You can look at other attributes for examples